sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/implementation/keyvault/environment/KeyVaultEnvironmentPostProcessorTests.java

```diff
@@ -43,16 +49,40 @@ class KeyVaultEnvironmentPostProcessorTests {
     private KeyVaultEnvironmentPostProcessor processor;
     private MockEnvironment environment;
     private MutablePropertySources propertySources;
+    private ConfigurableBootstrapContext context;
 
     @BeforeEach
     void beforeEach() {
+        context = mock(ConfigurableBootstrapContext.class);
```

Comment:
should we put these into the cases using the bootstrap context, instead of putting them in @BeforeEach section?